packages/manager/src/features/Lish/Lish.tsx

```diff
@@ -123,11 +126,11 @@ const Lish = () => {
       <TabLinkList tabs={tabs} />
       <TabPanels>
         <SafeTabPanel data-qa-tab="Weblish" index={0}>
-          <Weblish linode={linode} refreshToken={refreshToken} token={token} />
+          <Weblish linode={linode} refreshToken={refreshToken} weblish_url={weblish_url} ws_protocols={ws_protocols} />
```

Comment:
Then just pass in data like
```suggestion
          <Weblish linode={linode} refreshToken={refreshToken} {...data} />
```